data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,166 @@
+  accessing running or paused containers, all filesystem changes are directly
+  visible to the container. The `/nix` directory is never visible to the actual
+  image or container.
+  
+usage: debug [OPTIONS] {CONTAINER|IMAGE}
+pname: docker
+plink: docker.yaml
+options:
+    - option: shell
+      value_type: shell
+      default_value: "auto"
+      description: "Select a shell. Supported: `bash`, `fish`, `zsh`, `auto`."
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: command
+      shorthand: c
+      value_type: string
+      default_value: false
+      description: Evaluate the specified commands instead, passing additional positional arguments through $argv.
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: host
+      value_type: string
+      default_value: false
+      description: "Daemon docker socket to connect to. E.g.: `ssh://root@example.org`, `unix:///some/path/docker.sock`"
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: version
+      value_type: bool
+      default_value: "false"
+      description: Display version of the docker-debug plugin
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+deprecated: false
+experimental: false
+experimentalcli: false
+kubernetes: false
+swarm: false
+examples: |-
+  ### Example 1: Get a shell into any image
+  This example pulls an nginx image and gets a shell into it without ever running a container.
+  
+  ```console
+  $ docker debug nginx
+  
+  Builtin commands:
+  - install [tool1] [tool2] ...    Add Nix packages from: https://search.nixos.org/packages
+  - uninstall [tool1] [tool2] ...  Uninstall NixOS package(s).
+  - entrypoint                     Print/lint/run the entrypoint.
+  - builtins                       Show builtin commands.
+  
+  Checks:
+  ✓ distro:            Debian GNU/Linux 12 (bookworm)
+  ✓ entrypoint linter: no errors (run 'entrypoint' for details)
+
+  Note: This is a sandbox shell. All changes will not affect the actual image.
+                                                                                                                     Version: 0.0.18 (BETA)
+  root@a8758716bb6a / [nginx:latest]
+  docker >
+  ```
+  
+  The console now has a shell in the read-only filesystem of the nginx image.
+  You can now run `entrypoint` to inspect the entrypoint or run `builtins` to
+  see all builtin commands.
+  
+  ### Example 2: Dumping a file from an image to stdout by using the -c mode
+  
+  Use the `-c` (`--command`) flag to directly evaluate a command instead of
+  running the shell in interactive mode.
```

Comment:
Wondering if this should be working similar as  `--entrypoint` (like we have on `docker run` 🤔 ), which can disable the `/bin/sh` (or other entrypoint that the container / image may have; in `debug`'s case that entrypoint is the shell.

Or perhaps a cleaner approach would be to add a `none` option to the `--shell` flag;

```bash
docker debug --shell=none mycontainer /bin/some-command
```

Reply:
yeah, while it still was named `docker shell` this was way cleaner and clearer. It's just the equivalent of `bash/fish/zsh -c ARGS`. With `docker shell -c` that would have been clear directly, now we might need to explain..

Reply:
Added more examples to show the usage of all the options.